docs/book/component-gallery/model-registries/mlflow.md

```diff
@@ -0,0 +1,143 @@
+        )
+    ),
+).run()
+```
+
+### ZenML Command line interface (CLI)
+
+Sometimes adding a step to your pipeline is not the best option for you, as it
+will register the model in the MLflow Model Registry every time you run the
+pipeline. In this case, you can use the ZenML CLI to register your model
+manually. The CLI provides a command called `zenml model-registry models register-version`
+that you can use to register your model in the MLflow Model Registry.
+
+```shell
+zenml model-registry models register-version Tensorflow-model \
+    --model-uri="file:///.../mlruns/667102566783201219/3973eabc151c41e6ab98baeb20c5323b/artifacts/model" \
+    --tags key1 value1 --tags key2 value2 \
+    --description="A new version of the tensorflow model with accuracy 98.88%" \
+    --zenml-pipeline-name="mlflow_training_pipeline"
+```
+
+### List of available parameters
+
+To register a model version in the MLflow Model Registry, you need to provide
+list of parameters. when you use the built-in step, most of the parameters are
+automatically filled in for you. However, you can still override them if you
+want to. The following table shows the list of available parameters.
+
+* `name`: The name of the model. This is a required parameter.
+* `description`: A description of the registered model.
+* `tags`: A list of tags to associate with the registered model.
+* `model_uri`: The path to the model. This is a required parameter.
+* `version_description`: A description of the model version.
+* `version_tags`: A list of tags to associate with the model version.
+* `zenml_pipeline_name`: The name of the ZenML pipeline that produced the model.
+* `zenml_pipeline_run_id`: The run id of the ZenML pipeline that produced the
+model.
+* `zenml_pipeline_step_name`: The time when the ZenML pipeline that produced the
+model was run.
+
+{% hint style="info" %}
+The `model_uri` parameter is the path to the model within the MLflow tracking
+server. If you are using a local MLflow tracking server, the path will be
+something like `file:///.../mlruns/667102566783201219/3973eabc151c41e6ab98baeb20c5323b/artifacts/model`.
+If you are using a remote MLflow tracking server, the path will be something
+like `s3://.../mlruns/667102566783201219/3973eabc151c41e6ab98baeb20c5323b/artifacts/model`.
+
+You can find the path of the model in the MLflow UI. Go to the `Artifacts` tab
+of the run that produced the model and click on the model. The path will be
+displayed in the URL.
+
+![MLflow UI](../../assets/mlflow/mlflow_ui_uri.png)
+{% endhint %}
+
+Check out the
+[API docs](https://apidocs.zenml.io/latest/integration_code_docs/integrations-mlflow/#zenml.integrations.mlflow.model_registry.MLFlowModelRegistry)
+You can also check out our examples pages for working examples that use the
+MLflow Model Registry Example:
+
+- [Manage Models with MLflow](https://github.com/zenml-io/zenml/tree/main/examples/mlflow_registry)
```

Comment:
```suggestion
You can also [check out our examples page](https://github.com/zenml-io/zenml/tree/main/examples/mlflow_registry) for a working example that uses the
MLflow Model Registry.
```

When we have multiple we can change this to a list. Doesn't make sense right now.